MinPlatformPkg/PlatformInit/PlatformInitSmm/PlatformInitStandaloneMm.c

```diff
@@ -0,0 +1,153 @@
+/** @file
+  This driver will register two callbacks to call fsp's notifies.
```

Comment:
Like other file, please update this to say something about it being the file with Standalone MM specific functionality.

Reply:
Updated.